app/src/androidTest/java/com/mapbox/maps/testapp/viewport/ViewportPluginTest.kt

```diff
@@ -70,52 +68,135 @@ class ViewportPluginTest : BaseMapTest() {
 
   @Test
   fun transitionToDefaultTransition() {
-    assumeTrue(
-      "Can only run on API Level 24 or newer because of difference of animator behaviour.",
-      Build.VERSION.SDK_INT > 23
-    )
+    val latch = CountDownLatch(1)
     handler.post {
-      viewportPlugin.transitionTo(followPuckViewportState)
       assertEquals(0.0, mapView.getMapboxMap().cameraState.bearing, EPS)
       mapView.getMapboxMap().cameraState.center.assertEquals(NULL_ISLAND)
-      locationProvider.locationConsumers.forEach { it.onLocationUpdated(TEST_POINT) }
-      locationProvider.locationConsumers.forEach { it.onBearingUpdated(TEST_BEARING) }
+      // immediate update location puck to test location.
+      locationProvider.locationConsumers.forEach {
+        it.onLocationUpdated(
+          TEST_POINT,
+          options = { duration = 0 }
+        )
+      }
+      locationProvider.locationConsumers.forEach {
+        it.onBearingUpdated(
+          TEST_BEARING,
+          options = { duration = 0 }
+        )
+      }
+      // transition to the followPuckViewportState with default transition
+      viewportPlugin.transitionTo(followPuckViewportState) {
+        latch.countDown()
+      }
     }
 
-    val latch = CountDownLatch(1)
-    latch.await(4000, TimeUnit.MILLISECONDS)
+    // Wait for 5 seconds since the default transition time is 3.5 seconds
+    latch.await(5000, TimeUnit.SECONDS)
```

Comment:
Isn't this now waiting 5000 seconds?!?
```suggestion
    latch.await(5, TimeUnit.SECONDS)
```

Reply:
good catch 🙈